src/components/Table/Table.tsx

```diff
@@ -164,6 +164,7 @@ export const Table = ({
                                     <TableColumnHeader key={column.key} column={column} state={state} />
                                 ),
                             )}
+                            <td />
```

Comment:
I think that closing tag is not necessary.

Reply:
It is a self closing tag in order to simulate the last column header, otherwise the background doesn't fill the entire container for some reason.

Without: 
![image](https://user-images.githubusercontent.com/18708637/185603787-616b1b18-57a7-4296-bec4-90905ea40b4a.png)
 
With:
![image](https://user-images.githubusercontent.com/18708637/185603881-8ca6b8f4-b39b-4572-b633-8f36b6c39514.png)

Reply:
Oh I see! But that just doesn't seem right to have that `td` there. Is there really no better way?

Reply:
Actually it makes more sense to be an empty th! https://stackoverflow.com/a/19842702 Will update